### DISCUSSION TITLE
Is the suggested changes from sk-v1-proposal already in Beta8?

### DISCUSSION THREAD

**Discussion by R-Liebert** [microsoft/semantic-kernel#3708] [Q&A] [answered]
Hi, tried setting up kernel.RunAsync( .... , streaming: true); in my script. But it's not recognized. I'm using C# and have the beta8 package. The package has been updated since the feature was mentioned in the blog post, so was suprised it threw an error.

**Comment by evchaki** [comment, marked as answer]
@R-Liebert - check out the 1.0 release to see if this helps - https://learn.microsoft.com/en-us/dotnet/api/microsoft.semantickernel?view=semantic-kernel-dotnet
